src/plugins/settings/settings.tsx

```diff
@@ -22,28 +22,20 @@ export class UIPSettings extends UIPPlugin {
   @attr() public target: string;
   @attr({defaultValue: 'Settings'}) public label: string;
 
-  /** {@link ESLScrollbar} scroll element */
-  @memoize()
-  public get $scroll(): ESLScrollbar {
-    const $scroll = document.createElement('esl-scrollbar');
-    $scroll.setAttribute('target', '::prev(.settings-list)');
-    return $scroll;
-  }
-
   protected connectedCallback() {
     super.connectedCallback();
     this.updateInner();
   }
 
   /** Initializes settings layout */
   protected updateInner() {
-    const $settingsList = document.createElement('div');
-    $settingsList.className = 'settings-list esl-scrollable-content';
-    [...this.childNodes].forEach( (node: HTMLElement) => {
-      $settingsList.appendChild(node);
+    const $content = <div className="settings-list esl-scrollable-content">
+      <esl-scrollbar target="::prev(.settings-list)"></esl-scrollbar>
+    </div>;
+    [...this.childNodes].forEach((node: HTMLElement) => {
```

Comment:
There is a `childNodes.forEach` callback, so we don't need to create array here.